README.md

```diff
@@ -26,11 +26,12 @@ allprojects {
 }
 ```
 
-Add the dependency to your build.gradle
+The Yoti SDK is composed by different features. Each feature has its own dependency and therefore, you will only need to add the dependencies of the features you are interested in to your build.gradle:
```

Comment:
This could be a bit clearer, something like:

>The Yoti SDK is composed of multiple feature modules. Each feature is optional, but you must include at least one to use the SDK. The modules you include must match those requested by your backend. Attempts to use a module you haven't included will fail at runtime with a (600x response code)[#possible-status-for-the-session]. Add modules you require to your build.gradle: